### ISSUE TITLE
Always have the fill below the curve in area charts

### ISSUE THREAD

**Issue by jkohlin** [apexcharts/apexcharts.js#1472]
I'm creating trend line widgets, showing the angle of the current trend line (another request btw) for a given time period. Some chart values are however negative and that makes the chart "flip" the area or even worse, cross the line. Like in this codepen: 
https://codepen.io/jkohlin/pen/BaNgbOy 

What I want is an option for fill that might look like this:
```
fill: {
  colors: '#0000ff',
  opacity: 0.9,
  type: 'gradient',
  fillTo: -20,
}
```
Where `fillTo` is a numeric value that, if set, forces the fill to flow all the way down to that value on the y-axis. 
Then you could get this:
![image](https://user-images.githubusercontent.com/4112881/78562940-783d7d80-781a-11ea-9875-cc11121940bd.png)
instead of this: 
![image](https://user-images.githubusercontent.com/4112881/78563296-087bc280-781b-11ea-87c0-887e5749c2ed.png)

**Describe alternatives you've considered**	
At the moment I have to recalculate the y values and add a positive value big enough to force the curve above zero. This works on sparklines, where the y-axis is hidden, but not otherwise of course.

This would be an awesome feature!

**Comment by oherik** [comment]
I would, too, like to request this. As of now it looks rather strange for negative values, if the area has a gradient: 
![image](https://user-images.githubusercontent.com/10709871/82047622-653c8980-96b3-11ea-9bf6-858e18ea140d.png). It makes it quite difficult to see the lowest values.

**Comment by junedchhipa** [comment]
Sorry for the late response.

I have added a new option that will extend the area beneath zero-line and fill it till the end.

```javascript
plotOptions: {
  area: {
    fillTo: 'end'
  }
}
```
This will be available from v3.19.3
